### PR TITLE
ROX-20707: Update docs for location of watched images

### PR DESCRIPTION
**Pull request by kcarmichael08**
Version(s):

4.3+

[Issue](https://issues.redhat.com/browse/ROX-20707)

[Link to docs preview
](https://67639--docspreview.netlify.app/openshift-acs/latest/operating/examine-images-for-vulnerabilities#scan-inactive-images_examine-images-for-vulnerabilities)

QE review:
- [ ] QE has approved this change.

Additional information:

### CODE REVIEW

File: modules/scan-inactive-images.adoc [hunk 1]
```diff
@@ -11,24 +11,18 @@
...
 ====
-On the {product-title-short} portal, click *Platform Configuration > System Configuration* to view the data retention configuration.
+On the {product-title-short} portal, click *Platform Configuration* -> *System Configuration* to view the data retention configuration.
 
-All the data related to the image removed from the watchlist continues to appear in the {product-title-short} portal for the number of days mentioned on the *System Configuration* page and is only removed after that period is over.
+All the data related to the image removed from the watched image list continues to appear in the {product-title-short} portal for the number of days mentioned on the *System Configuration* page and is only removed after that period is over.
 ====
-. Select *RETURN TO IMAGE LIST* to view the *IMAGES* page.
\ No newline at end of file
+. Click *Closed* to return to the *Workload CVEs* page.
```
Comment (dvail) [inline modules/scan-inactive-images.adoc:28]:
```suggestion
. Click *Close* to return to the *Workload CVEs* page.
```

**Review by dvail — APPROVED** [review]
LGTM :shipit:

**Review by skrthomas — APPROVED** [review]
just one small nit, otherwise lgtm!

**Pull request merged**